module1/task2.py

```diff
@@ -0,0 +1,12 @@
+numbers = [1, 2, '0', '300', -2.5, 'Dog', True, 0o1256, None]
+max_value = numbers[0]
+min_value = numbers[0]
+for n in numbers:
+    try:
+        temp = int(n)
+        max_value = max(max_value, temp)
+        min_value = min(min_value, temp)
+    except ValueError:
+        print(f"It is not possible to convert {n} to int")
+print(f"Max value from the list is {max_value}")
+print(f"Min value from the list is {min_value}")
```

Comment:
Program doesn't run in the way it looks so far

int() argument must be a string, a bytes-like object or a number, not 'NoneType'

Overall implementation looks fine

Reply:
Sorry, my bad, fixed